cmd/kube-apiserver/app/server.go

```diff
@@ -556,7 +565,9 @@ func (s *APIServer) Run(_ []string) error {
 		Authorizer:                authorizer,
 		AdmissionControl:          admissionController,
 		DisableV1:                 disableV1,
-		EnableExp:                 enableExp,
+		EnableExtensions:          enableExtensions,
```

Comment:
Have added a generic apiGroupVersionOverrides.
But have not added support for enabling/disabling individual v1 resources. I dont think we need it. Can add it later, if required.